laboratory/resources/web/laboratory/panel/AbstractAssayPanel.js

```diff
@@ -197,6 +197,17 @@ Ext4.define('Laboratory.panel.AbstractAssayPanel', {
             }
         }
 
+        if (fieldObj.getInitialValue){
```

Comment:
Doesnt this addition just duplicate the code right above it?

Reply:
Yup, now removed - looks like it happened as part of 17.2 merges to wnprc18.3 branch.